src/modules/controllers/task.controller.js

```diff
@@ -0,0 +1,76 @@
+const Task = require('../../models/task');
+
+const getAllTasks = async (req, res) => {
+  try {
+    const result = await Task.find();
+    res.status(200).send({ data: result })
+  } catch (error) {
+    res.status(400).send('Bad Request tasks not found')
+    console.error('tasks not found');
+  }
+}
+
+const createNewTask = async (req, res) => {
+  try {
+    const task = new Task(req.body);
```

Comment:
don't take "isCheck" from the frontend. Create it on the default backend "false"